### PR TITLE
Rename epang to epang/place

### PR DESCRIPTION
**Pull request by erikrikarddaniel**
## PR checklist

- [x] This comment contains a description of changes (with reason).
- [] If you've fixed a bug or added code that should be tested, add tests!
- [] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [ ] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

I renamed the `epang` module to `epang/place` to make place for a `epang/split` module, see https://nfcore.slack.com/archives/CJRH30T6V/p1671109549454979 for a discussion.

### CODE REVIEW

**Review by nvnieuwk — COMMENTED** [review]
Can you also update the `pytest_modules.yml` file to follow the new naming?

**Comment by erikrikarddaniel** [comment]
> Can you also update the `pytest_modules.yml` file to follow the new naming?

Done. (Forgot the linting step, seems to work now.)

**Review by nvnieuwk — APPROVED** [review]
LGTM! Maybe wait for another review as I'm not familiar with the tool

**Pull request merged**
